src/System.Buffers/src/System.Buffers.builds

```diff
@@ -2,7 +2,12 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Buffers.csproj" />
+    <Project Include="System.Buffers.csproj">
+      <TargetGroup>netstandard1.1</TargetGroup>
+    </Project>
+    <Project Include="System.Buffers.csproj">
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
`netcoreapp1.1` is the default everywhere else. Any reason why this should be different?